src/xinterpreter.cpp

```diff
@@ -253,7 +253,9 @@ namespace xpyt
         };
 
         m_displayhook.attr("add_hook")(publish_display);
-
         sys.attr("displayhook") = m_displayhook;
+
+        // Create a global "display" function
+        py::globals()["display"] = m_displayhook;
```

Comment:
I guess it will be more complicated than that. Because the `display` function from `IPython` takes some arguments that could be useful in some cases?